tests/run-tests.sh

```
@@ -477,4 +477,13 @@ print_info "Running unified workflow tests..."
 bash ./tests/test-unified-workflow.sh
 print_success "Unified workflow tests passed."
 
+# Test 12: Loadfile-only and Chaos Engine tests
+print_info "Running loadfile-only and Chaos Engine tests..."
+bash ./tests/run-e2e-loadfile.sh
+if [ $? -ne 0 ]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?.

(SC2181)